tests/phpunit/api/v3/NavigationTest.php

```diff
@@ -46,7 +46,7 @@ public function testGet() {
    * Test create function.
    */
   public function testCreate() {
-    $params = array('label' => 'Feed the Goats', 'domain_id' => 1);
+    $params = array('name' => 'feed', 'label' => 'Feed the Goats', 'domain_id' => 1);
```

Comment:
Or did I make the wrong param required? Does the code only actually require either/or? (in which case validate_one_mandatory makes sense